tests/rptest/services/redpanda.py

```diff
@@ -4226,25 +4238,31 @@ def _update_usage_stats(self, node: ClusterNode):
         def _metrics_sum(name: str) -> int:
             try:
                 samples = self.metrics_sample(name, [node])
-
                 if samples is None:
                     return 0
-
                 return sum(s.value for s in samples.samples)
             except Exception as e:
                 self.logger.warning(f"Cannot check metrics - {e}")
                 return 0
 
-        self._usage_stats.bytes_read += _metrics_sum(
+        self._usage_stats.disk_bytes_read += _metrics_sum(
             "vectorized_io_queue_total_read_bytes_total")
 
-        self._usage_stats.bytes_written += _metrics_sum(
+        self._usage_stats.disk_bytes_written += _metrics_sum(
             "vectorized_io_queue_total_write_bytes_total")
         self._usage_stats.batches_read += _metrics_sum(
             "vectorized_storage_log_batches_read")
 
         self._usage_stats.batches_written += _metrics_sum(
             "vectorized_storage_log_batches_written")
+        self._usage_stats.internal_rpc_bytes_sent += _metrics_sum(
```

Comment:
maybe add a quick comment here explaining that the seemingly wrong way around metrics is actually correct as it's measured at the server.

Reply:
Sorry can you clarify? I am not following

Reply:
See you have made it match now. Just want to be clear again, in practice what you will see is that `vectorized_internal_rpc_received_bytes` is the large number (i.e.: matching produce) because it's measured at the receiving server side.

Think either fine is fine but just wanted to be clear because this is often confusing.